src/solver/simulation/economy.cpp

```diff
@@ -106,12 +105,12 @@ weeklyOptimization::weeklyOptimization(PROBLEME_HEBDO** problemesHebdo) :
  interfaceWeeklyOptimization(problemesHebdo)
 {
 }
-void weeklyOptimization::solve(Variable::State&, int, uint numSpace, uint)
+void weeklyOptimization::solve(uint, int, uint numSpace)
 {
     auto problemeHebdo = pProblemesHebdo[numSpace];
     OPT_OptimisationHebdomadaire(problemeHebdo, numSpace);
 }
-void weeklyOptimization::solveCSR(const Variable::State& state, uint numSpace, uint week)
+void weeklyOptimization::solveCSR(Antares::Data::AreaList&, uint, uint, uint)
 {
     return;
 }
```

Comment:
No need for an abstract method. Use an empty method in the base class

Reply:
Done